I 03-17 21:19:38 controller.py:59] DAG:
I 03-17 21:19:38 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 21:19:38 controller.py:59]   resources: {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}]
I 03-17 21:19:38 controller.py:195] Submitted managed job 199 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-21-19-38-714666_compute-text-vectors_199-0
I 03-17 21:19:38 controller.py:199] Started monitoring.
I 03-17 21:19:38 state.py:451] Launching the spot cluster...
I 03-17 21:19:39 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 21:19:39 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:19:39 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:19:39 optimizer.py:749] 
I 03-17 21:19:39 optimizer.py:884] Considered resources (1 node):
I 03-17 21:19:39 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:19:39 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:19:39 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:19:39 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 21:19:39 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:19:39 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:19:40 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
W 03-17 21:32:00 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:32:00 cloud_vm_ray_backend.py:2097] 
W 03-17 21:32:00 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:32:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:32:01 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:32:01 optimizer.py:749] 
I 03-17 21:32:01 optimizer.py:884] Considered resources (1 node):
I 03-17 21:32:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:32:01 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:32:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:32:01 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 21:32:01 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:32:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:32:01 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
[2mW 03-17 21:39:14 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:39:15 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:39:15 cloud_vm_ray_backend.py:2097] 
W 03-17 21:39:15 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:39:15 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:39:15 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:39:15 optimizer.py:749] 
I 03-17 21:39:15 optimizer.py:884] Considered resources (1 node):
I 03-17 21:39:15 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:39:15 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:39:15 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:39:15 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 21:39:15 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:39:15 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:39:15 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 21:40:45 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2d). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2a.[0m
W 03-17 21:40:46 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:40:46 cloud_vm_ray_backend.py:2097] 
W 03-17 21:40:46 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:40:46 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:40:46 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:40:46 optimizer.py:749] 
I 03-17 21:40:46 optimizer.py:884] Considered resources (1 node):
I 03-17 21:40:46 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:40:46 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:40:46 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:40:46 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 21:40:46 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:40:46 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:40:46 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
W 03-17 21:49:46 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:49:46 cloud_vm_ray_backend.py:2097] 
W 03-17 21:49:46 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:49:47 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:49:47 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:49:47 optimizer.py:749] 
I 03-17 21:49:47 optimizer.py:884] Considered resources (1 node):
I 03-17 21:49:47 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:49:47 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:49:47 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:49:47 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 21:49:47 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:49:47 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:49:47 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
[2mW 03-17 21:49:53 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-1a, us-east-1c, us-east-1d.[0m
W 03-17 21:49:53 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:49:53 cloud_vm_ray_backend.py:2097] 
W 03-17 21:49:53 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:49:54 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:49:54 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:49:54 optimizer.py:749] 
I 03-17 21:49:54 optimizer.py:884] Considered resources (1 node):
I 03-17 21:49:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:49:54 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:49:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:49:54 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 21:49:54 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:49:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:49:54 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 21:50:00 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 21:50:01 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:01 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:01 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:01 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:50:01 optimizer.py:749] 
I 03-17 21:50:01 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:01 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:50:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:01 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 21:50:01 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:50:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:01 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 21:50:12 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:50:13 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:13 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:13 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:13 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:13 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:50:13 optimizer.py:749] 
I 03-17 21:50:13 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:13 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:50:13 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 21:50:13 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:50:13 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 21:50:13 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 21:50:13 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:13 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 21:50:18 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:50:18 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:18 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:18 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:19 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:50:19 optimizer.py:749] 
I 03-17 21:50:19 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:19 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:50:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:19 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 21:50:19 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:50:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:19 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 21:50:20 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 21:50:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:20 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:20 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:50:20 optimizer.py:749] 
I 03-17 21:50:20 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:20 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:50:20 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 21:50:20 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:50:20 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 21:50:20 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 21:50:20 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:21 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 21:50:25 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:50:26 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:26 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:26 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:26 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:26 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:50:26 optimizer.py:749] 
I 03-17 21:50:26 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:26 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:26 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:50:26 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:26 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 21:50:26 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:50:26 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:26 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 21:50:33 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:50:34 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:34 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:34 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:34 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:34 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:50:34 optimizer.py:749] 
I 03-17 21:50:34 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:34 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:34 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:50:34 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:34 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 21:50:34 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:50:34 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:35 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
[2mW 03-17 21:50:43 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:50:44 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:44 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:44 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:44 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:44 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:50:44 optimizer.py:749] 
I 03-17 21:50:44 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:44 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:44 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:50:44 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:44 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1a   0.35       [32m   âœ”[0m     
I 03-17 21:50:44 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:50:44 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:44 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a).
[2mW 03-17 21:50:56 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:50:57 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:50:57 cloud_vm_ray_backend.py:2097] 
W 03-17 21:50:57 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:50:57 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:50:57 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:50:57 optimizer.py:749] 
I 03-17 21:50:57 optimizer.py:884] Considered resources (1 node):
I 03-17 21:50:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:57 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:50:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:50:57 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2b    0.36       [32m   âœ”[0m     
I 03-17 21:50:57 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:50:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:50:57 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2b).
[2mW 03-17 21:51:04 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:51:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:51:05 cloud_vm_ray_backend.py:2097] 
W 03-17 21:51:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:51:05 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:51:05 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:51:05 optimizer.py:749] 
I 03-17 21:51:05 optimizer.py:884] Considered resources (1 node):
I 03-17 21:51:05 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:51:05 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:51:05 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:51:05 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2c   0.36       [32m   âœ”[0m     
I 03-17 21:51:05 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:51:05 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:51:06 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2c).
[2mW 03-17 21:51:18 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:51:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:51:19 cloud_vm_ray_backend.py:2097] 
W 03-17 21:51:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:51:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:51:19 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:51:19 optimizer.py:749] 
I 03-17 21:51:19 optimizer.py:884] Considered resources (1 node):
I 03-17 21:51:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:51:19 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:51:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:51:19 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1d    0.37       [32m   âœ”[0m     
I 03-17 21:51:19 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:51:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:51:19 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1d).
[2mW 03-17 21:51:31 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:51:31 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1d for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:51:31 cloud_vm_ray_backend.py:2097] 
W 03-17 21:51:31 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:51:32 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:51:32 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:51:32 optimizer.py:749] 
I 03-17 21:51:32 optimizer.py:884] Considered resources (1 node):
I 03-17 21:51:32 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:51:32 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:51:32 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:51:32 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2b    0.37       [32m   âœ”[0m     
I 03-17 21:51:32 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:51:32 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:51:32 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2b).
[2mW 03-17 21:51:39 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2a, us-west-2c, us-west-2d.[0m
W 03-17 21:51:40 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:51:40 cloud_vm_ray_backend.py:2097] 
W 03-17 21:51:40 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:51:40 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:51:40 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:51:40 optimizer.py:749] 
I 03-17 21:51:40 optimizer.py:884] Considered resources (1 node):
I 03-17 21:51:40 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:51:40 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:51:40 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:51:40 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1b   0.37       [32m   âœ”[0m     
I 03-17 21:51:40 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:51:40 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:51:40 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1b).
[2mW 03-17 21:51:51 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:51:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:51:52 cloud_vm_ray_backend.py:2097] 
W 03-17 21:51:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:51:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:51:52 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:51:52 optimizer.py:749] 
I 03-17 21:51:52 optimizer.py:884] Considered resources (1 node):
I 03-17 21:51:52 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:51:52 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:51:52 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:51:52 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2a   0.38       [32m   âœ”[0m     
I 03-17 21:51:52 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:51:52 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:51:52 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2a).
[2mW 03-17 21:52:04 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:52:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:52:05 cloud_vm_ray_backend.py:2097] 
W 03-17 21:52:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:52:05 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:52:05 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:52:05 optimizer.py:749] 
I 03-17 21:52:05 optimizer.py:884] Considered resources (1 node):
I 03-17 21:52:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:52:05 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:52:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:52:05 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3b    0.38       [32m   âœ”[0m     
I 03-17 21:52:05 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:52:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:52:05 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b).
[2mW 03-17 21:52:13 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:52:14 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:52:14 cloud_vm_ray_backend.py:2097] 
W 03-17 21:52:14 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:52:14 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:52:14 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:52:14 optimizer.py:749] 
I 03-17 21:52:14 optimizer.py:884] Considered resources (1 node):
I 03-17 21:52:14 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:52:14 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:52:14 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:52:14 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1a    0.39       [32m   âœ”[0m     
I 03-17 21:52:14 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:52:14 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:52:14 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a).
[2mW 03-17 21:52:55 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:52:56 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:52:56 cloud_vm_ray_backend.py:2097] 
W 03-17 21:52:56 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:52:56 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:52:56 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:52:56 optimizer.py:749] 
I 03-17 21:52:56 optimizer.py:884] Considered resources (1 node):
I 03-17 21:52:56 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:52:56 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:52:56 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:52:56 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2a    0.40       [32m   âœ”[0m     
I 03-17 21:52:56 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:52:56 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:52:56 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a).
[2mW 03-17 21:53:20 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:53:21 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:53:21 cloud_vm_ray_backend.py:2097] 
W 03-17 21:53:21 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:53:21 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:53:21 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:53:21 optimizer.py:749] 
I 03-17 21:53:21 optimizer.py:884] Considered resources (1 node):
I 03-17 21:53:21 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:53:21 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 21:53:21 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:53:21 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1b   0.40       [32m   âœ”[0m     
I 03-17 21:53:21 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 21:53:21 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:53:21 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1b).
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:53:50 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:54:17 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-199.[0m  [2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 21:54:17 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:54:17 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-2qjgobb6/job-2qjgobb6/workdir -> ~/sky_workdir[0m
I 03-17 21:54:21 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:54:21 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final-2 -> /output[0m
I 03-17 21:54:23 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-21-19-39-350474/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 21:54:25 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 21:54:26 cloud_vm_ray_backend.py:3546] Multiple resources are specified for the task, using: AWS([Spot], cpus=4, {'L4': 1})
I 03-17 21:54:27 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 21:54:28 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 21:54:31 utils.py:131] === Checking the job status... ===
I 03-17 21:54:32 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 21:54:32 utils.py:140] ==================================
I 03-17 21:54:33 state.py:471] Job started.
I 03-17 21:54:53 utils.py:131] === Checking the job status... ===
I 03-17 21:54:54 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 21:54:54 utils.py:140] ==================================
I 03-17 21:55:14 utils.py:131] === Checking the job status... ===
I 03-17 21:55:15 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 21:55:15 utils.py:140] ==================================
I 03-17 21:55:35 utils.py:131] === Checking the job status... ===
I 03-17 21:55:36 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 21:55:36 utils.py:140] ==================================
I 03-17 21:55:56 utils.py:131] === Checking the job status... ===
I 03-17 21:55:57 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 21:55:57 utils.py:140] ==================================
I 03-17 21:56:17 utils.py:131] === Checking the job status... ===
I 03-17 21:56:18 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 21:56:18 utils.py:140] ==================================
I 03-17 21:56:38 utils.py:131] === Checking the job status... ===
I 03-17 21:56:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:56:39 utils.py:140] ==================================
I 03-17 21:56:59 utils.py:131] === Checking the job status... ===
I 03-17 21:57:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:57:01 utils.py:140] ==================================
I 03-17 21:57:21 utils.py:131] === Checking the job status... ===
I 03-17 21:57:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:57:22 utils.py:140] ==================================
I 03-17 21:57:42 utils.py:131] === Checking the job status... ===
I 03-17 21:57:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:57:43 utils.py:140] ==================================
I 03-17 21:58:03 utils.py:131] === Checking the job status... ===
I 03-17 21:58:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:58:04 utils.py:140] ==================================
I 03-17 21:58:24 utils.py:131] === Checking the job status... ===
I 03-17 21:58:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:58:25 utils.py:140] ==================================
I 03-17 21:58:45 utils.py:131] === Checking the job status... ===
I 03-17 21:58:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:58:46 utils.py:140] ==================================
I 03-17 21:59:06 utils.py:131] === Checking the job status... ===
I 03-17 21:59:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:59:07 utils.py:140] ==================================
I 03-17 21:59:27 utils.py:131] === Checking the job status... ===
I 03-17 21:59:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:59:28 utils.py:140] ==================================
I 03-17 21:59:48 utils.py:131] === Checking the job status... ===
I 03-17 21:59:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 21:59:49 utils.py:140] ==================================
I 03-17 22:00:09 utils.py:131] === Checking the job status... ===
I 03-17 22:00:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:00:10 utils.py:140] ==================================
I 03-17 22:00:30 utils.py:131] === Checking the job status... ===
I 03-17 22:00:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:00:33 utils.py:140] ==================================
I 03-17 22:00:54 utils.py:131] === Checking the job status... ===
I 03-17 22:00:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:00:57 utils.py:140] ==================================
I 03-17 22:01:17 utils.py:131] === Checking the job status... ===
I 03-17 22:01:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:01:19 utils.py:140] ==================================
I 03-17 22:01:39 utils.py:131] === Checking the job status... ===
I 03-17 22:01:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:01:40 utils.py:140] ==================================
I 03-17 22:02:00 utils.py:131] === Checking the job status... ===
I 03-17 22:02:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:02:00 utils.py:140] ==================================
I 03-17 22:02:21 utils.py:131] === Checking the job status... ===
I 03-17 22:02:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:02:21 utils.py:140] ==================================
I 03-17 22:02:41 utils.py:131] === Checking the job status... ===
I 03-17 22:02:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:02:42 utils.py:140] ==================================
I 03-17 22:03:02 utils.py:131] === Checking the job status... ===
I 03-17 22:03:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:03:03 utils.py:140] ==================================
I 03-17 22:03:23 utils.py:131] === Checking the job status... ===
I 03-17 22:03:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:03:24 utils.py:140] ==================================
I 03-17 22:03:44 utils.py:131] === Checking the job status... ===
I 03-17 22:03:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:03:45 utils.py:140] ==================================
I 03-17 22:04:05 utils.py:131] === Checking the job status... ===
I 03-17 22:04:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:04:06 utils.py:140] ==================================
I 03-17 22:04:26 utils.py:131] === Checking the job status... ===
I 03-17 22:04:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:04:27 utils.py:140] ==================================
I 03-17 22:04:47 utils.py:131] === Checking the job status... ===
I 03-17 22:04:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:04:48 utils.py:140] ==================================
I 03-17 22:05:08 utils.py:131] === Checking the job status... ===
I 03-17 22:05:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:05:09 utils.py:140] ==================================
I 03-17 22:05:29 utils.py:131] === Checking the job status... ===
I 03-17 22:05:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:05:30 utils.py:140] ==================================
I 03-17 22:05:50 utils.py:131] === Checking the job status... ===
I 03-17 22:05:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:05:51 utils.py:140] ==================================
I 03-17 22:06:11 utils.py:131] === Checking the job status... ===
I 03-17 22:06:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:06:12 utils.py:140] ==================================
I 03-17 22:06:32 utils.py:131] === Checking the job status... ===
I 03-17 22:06:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:06:32 utils.py:140] ==================================
I 03-17 22:06:52 utils.py:131] === Checking the job status... ===
I 03-17 22:06:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:06:53 utils.py:140] ==================================
I 03-17 22:07:13 utils.py:131] === Checking the job status... ===
I 03-17 22:07:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:07:14 utils.py:140] ==================================
I 03-17 22:07:34 utils.py:131] === Checking the job status... ===
I 03-17 22:07:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:07:35 utils.py:140] ==================================
I 03-17 22:07:55 utils.py:131] === Checking the job status... ===
I 03-17 22:07:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:07:56 utils.py:140] ==================================
I 03-17 22:08:16 utils.py:131] === Checking the job status... ===
I 03-17 22:08:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:08:17 utils.py:140] ==================================
I 03-17 22:08:37 utils.py:131] === Checking the job status... ===
I 03-17 22:08:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:08:38 utils.py:140] ==================================
I 03-17 22:08:58 utils.py:131] === Checking the job status... ===
I 03-17 22:08:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:08:59 utils.py:140] ==================================
I 03-17 22:09:19 utils.py:131] === Checking the job status... ===
I 03-17 22:09:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:09:19 utils.py:140] ==================================
I 03-17 22:09:40 utils.py:131] === Checking the job status... ===
I 03-17 22:09:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:09:40 utils.py:140] ==================================
I 03-17 22:10:01 utils.py:131] === Checking the job status... ===
I 03-17 22:10:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:10:01 utils.py:140] ==================================
I 03-17 22:10:21 utils.py:131] === Checking the job status... ===
I 03-17 22:10:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:10:22 utils.py:140] ==================================
I 03-17 22:10:42 utils.py:131] === Checking the job status... ===
I 03-17 22:10:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:10:43 utils.py:140] ==================================
I 03-17 22:11:03 utils.py:131] === Checking the job status... ===
I 03-17 22:11:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:11:04 utils.py:140] ==================================
I 03-17 22:11:24 utils.py:131] === Checking the job status... ===
I 03-17 22:11:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:11:25 utils.py:140] ==================================
I 03-17 22:11:45 utils.py:131] === Checking the job status... ===
I 03-17 22:11:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:11:46 utils.py:140] ==================================
I 03-17 22:12:06 utils.py:131] === Checking the job status... ===
I 03-17 22:12:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:12:07 utils.py:140] ==================================
I 03-17 22:12:27 utils.py:131] === Checking the job status... ===
I 03-17 22:12:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:12:28 utils.py:140] ==================================
I 03-17 22:12:48 utils.py:131] === Checking the job status... ===
I 03-17 22:12:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:12:49 utils.py:140] ==================================
I 03-17 22:13:09 utils.py:131] === Checking the job status... ===
I 03-17 22:13:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:13:10 utils.py:140] ==================================
I 03-17 22:13:30 utils.py:131] === Checking the job status... ===
I 03-17 22:13:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:13:31 utils.py:140] ==================================
I 03-17 22:13:51 utils.py:131] === Checking the job status... ===
I 03-17 22:13:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:13:51 utils.py:140] ==================================
I 03-17 22:14:12 utils.py:131] === Checking the job status... ===
I 03-17 22:14:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:14:12 utils.py:140] ==================================
I 03-17 22:14:33 utils.py:131] === Checking the job status... ===
I 03-17 22:14:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:14:33 utils.py:140] ==================================
I 03-17 22:14:53 utils.py:131] === Checking the job status... ===
I 03-17 22:14:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:14:54 utils.py:140] ==================================
I 03-17 22:15:14 utils.py:131] === Checking the job status... ===
I 03-17 22:15:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:15:15 utils.py:140] ==================================
I 03-17 22:15:35 utils.py:131] === Checking the job status... ===
I 03-17 22:15:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:15:36 utils.py:140] ==================================
I 03-17 22:15:56 utils.py:131] === Checking the job status... ===
I 03-17 22:15:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:15:57 utils.py:140] ==================================
I 03-17 22:16:17 utils.py:131] === Checking the job status... ===
I 03-17 22:16:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:16:18 utils.py:140] ==================================
I 03-17 22:16:38 utils.py:131] === Checking the job status... ===
I 03-17 22:16:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:16:39 utils.py:140] ==================================
I 03-17 22:16:59 utils.py:131] === Checking the job status... ===
I 03-17 22:17:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:17:00 utils.py:140] ==================================
I 03-17 22:17:20 utils.py:131] === Checking the job status... ===
I 03-17 22:17:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:17:20 utils.py:140] ==================================
I 03-17 22:17:41 utils.py:131] === Checking the job status... ===
I 03-17 22:17:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:17:41 utils.py:140] ==================================
I 03-17 22:18:02 utils.py:131] === Checking the job status... ===
I 03-17 22:18:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:18:02 utils.py:140] ==================================
I 03-17 22:18:22 utils.py:131] === Checking the job status... ===
I 03-17 22:18:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:18:23 utils.py:140] ==================================
I 03-17 22:18:43 utils.py:131] === Checking the job status... ===
I 03-17 22:18:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:18:44 utils.py:140] ==================================
I 03-17 22:19:04 utils.py:131] === Checking the job status... ===
I 03-17 22:19:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:19:05 utils.py:140] ==================================
I 03-17 22:19:25 utils.py:131] === Checking the job status... ===
I 03-17 22:19:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:19:26 utils.py:140] ==================================
I 03-17 22:19:46 utils.py:131] === Checking the job status... ===
I 03-17 22:19:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:19:47 utils.py:140] ==================================
I 03-17 22:20:07 utils.py:131] === Checking the job status... ===
I 03-17 22:20:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:20:08 utils.py:140] ==================================
I 03-17 22:20:28 utils.py:131] === Checking the job status... ===
I 03-17 22:20:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:20:29 utils.py:140] ==================================
I 03-17 22:20:49 utils.py:131] === Checking the job status... ===
I 03-17 22:20:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:20:50 utils.py:140] ==================================
I 03-17 22:21:10 utils.py:131] === Checking the job status... ===
I 03-17 22:21:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:21:10 utils.py:140] ==================================
I 03-17 22:21:31 utils.py:131] === Checking the job status... ===
I 03-17 22:21:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:21:31 utils.py:140] ==================================
I 03-17 22:21:51 utils.py:131] === Checking the job status... ===
I 03-17 22:21:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:21:52 utils.py:140] ==================================
I 03-17 22:22:12 utils.py:131] === Checking the job status... ===
I 03-17 22:22:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:22:13 utils.py:140] ==================================
I 03-17 22:22:33 utils.py:131] === Checking the job status... ===
I 03-17 22:22:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:22:34 utils.py:140] ==================================
I 03-17 22:22:54 utils.py:131] === Checking the job status... ===
I 03-17 22:22:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:22:55 utils.py:140] ==================================
I 03-17 22:23:15 utils.py:131] === Checking the job status... ===
I 03-17 22:23:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:23:16 utils.py:140] ==================================
I 03-17 22:23:36 utils.py:131] === Checking the job status... ===
I 03-17 22:23:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:23:37 utils.py:140] ==================================
I 03-17 22:23:57 utils.py:131] === Checking the job status... ===
I 03-17 22:23:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:23:58 utils.py:140] ==================================
I 03-17 22:24:18 utils.py:131] === Checking the job status... ===
I 03-17 22:24:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:24:19 utils.py:140] ==================================
I 03-17 22:24:39 utils.py:131] === Checking the job status... ===
I 03-17 22:24:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:24:40 utils.py:140] ==================================
I 03-17 22:25:00 utils.py:131] === Checking the job status... ===
I 03-17 22:25:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:25:00 utils.py:140] ==================================
I 03-17 22:25:21 utils.py:131] === Checking the job status... ===
I 03-17 22:25:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:25:21 utils.py:140] ==================================
I 03-17 22:25:41 utils.py:131] === Checking the job status... ===
I 03-17 22:25:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:25:42 utils.py:140] ==================================
I 03-17 22:26:02 utils.py:131] === Checking the job status... ===
I 03-17 22:26:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:26:03 utils.py:140] ==================================
I 03-17 22:26:23 utils.py:131] === Checking the job status... ===
I 03-17 22:26:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:26:24 utils.py:140] ==================================
I 03-17 22:26:44 utils.py:131] === Checking the job status... ===
I 03-17 22:26:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:26:45 utils.py:140] ==================================
I 03-17 22:27:05 utils.py:131] === Checking the job status... ===
I 03-17 22:27:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:27:06 utils.py:140] ==================================
I 03-17 22:27:26 utils.py:131] === Checking the job status... ===
I 03-17 22:27:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:27:27 utils.py:140] ==================================
I 03-17 22:27:47 utils.py:131] === Checking the job status... ===
I 03-17 22:27:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:27:48 utils.py:140] ==================================
I 03-17 22:28:08 utils.py:131] === Checking the job status... ===
I 03-17 22:28:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:28:09 utils.py:140] ==================================
I 03-17 22:28:29 utils.py:131] === Checking the job status... ===
I 03-17 22:28:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:28:30 utils.py:140] ==================================
I 03-17 22:28:50 utils.py:131] === Checking the job status... ===
I 03-17 22:28:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:28:50 utils.py:140] ==================================
I 03-17 22:29:11 utils.py:131] === Checking the job status... ===
I 03-17 22:29:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:29:11 utils.py:140] ==================================
I 03-17 22:29:32 utils.py:131] === Checking the job status... ===
I 03-17 22:29:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:29:32 utils.py:140] ==================================
I 03-17 22:29:52 utils.py:131] === Checking the job status... ===
I 03-17 22:29:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:29:53 utils.py:140] ==================================
I 03-17 22:30:13 utils.py:131] === Checking the job status... ===
I 03-17 22:30:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:30:14 utils.py:140] ==================================
I 03-17 22:30:34 utils.py:131] === Checking the job status... ===
I 03-17 22:30:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:30:35 utils.py:140] ==================================
I 03-17 22:30:55 utils.py:131] === Checking the job status... ===
I 03-17 22:30:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:30:56 utils.py:140] ==================================
I 03-17 22:31:16 utils.py:131] === Checking the job status... ===
I 03-17 22:31:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:31:17 utils.py:140] ==================================
I 03-17 22:31:37 utils.py:131] === Checking the job status... ===
I 03-17 22:31:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:31:38 utils.py:140] ==================================
I 03-17 22:31:58 utils.py:131] === Checking the job status... ===
I 03-17 22:31:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:31:59 utils.py:140] ==================================
I 03-17 22:32:19 utils.py:131] === Checking the job status... ===
I 03-17 22:32:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:32:20 utils.py:140] ==================================
I 03-17 22:32:40 utils.py:131] === Checking the job status... ===
I 03-17 22:32:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:32:40 utils.py:140] ==================================
I 03-17 22:33:01 utils.py:131] === Checking the job status... ===
I 03-17 22:33:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:33:01 utils.py:140] ==================================
I 03-17 22:33:21 utils.py:131] === Checking the job status... ===
I 03-17 22:33:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:33:22 utils.py:140] ==================================
I 03-17 22:33:42 utils.py:131] === Checking the job status... ===
I 03-17 22:33:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:33:43 utils.py:140] ==================================
I 03-17 22:34:03 utils.py:131] === Checking the job status... ===
I 03-17 22:34:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:34:04 utils.py:140] ==================================
I 03-17 22:34:24 utils.py:131] === Checking the job status... ===
I 03-17 22:34:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:34:25 utils.py:140] ==================================
I 03-17 22:34:45 utils.py:131] === Checking the job status... ===
I 03-17 22:34:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:34:46 utils.py:140] ==================================
I 03-17 22:35:06 utils.py:131] === Checking the job status... ===
I 03-17 22:35:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:35:07 utils.py:140] ==================================
I 03-17 22:35:27 utils.py:131] === Checking the job status... ===
I 03-17 22:35:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:35:28 utils.py:140] ==================================
I 03-17 22:35:48 utils.py:131] === Checking the job status... ===
I 03-17 22:35:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:35:48 utils.py:140] ==================================
I 03-17 22:36:09 utils.py:131] === Checking the job status... ===
I 03-17 22:36:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:36:09 utils.py:140] ==================================
I 03-17 22:36:29 utils.py:131] === Checking the job status... ===
I 03-17 22:36:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:36:30 utils.py:140] ==================================
I 03-17 22:36:50 utils.py:131] === Checking the job status... ===
I 03-17 22:36:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:36:51 utils.py:140] ==================================
I 03-17 22:37:11 utils.py:131] === Checking the job status... ===
I 03-17 22:37:12 utils.py:137] Job status: JobStatus.SUCCEEDED
I 03-17 22:37:12 utils.py:140] ==================================
I 03-17 22:37:13 state.py:561] Job succeeded.
I 03-17 22:37:13 controller.py:238] Managed job 199 (task: 0) SUCCEEDED. Cleaning up the cluster compute-text-vectors-199.
E 03-17 22:37:15 controller_utils.py:376] Failed to find the logs for the user program at ~/sky_logs/managed_jobs/sky-2025-03-17-21-19-39-350474/run.log.
I 03-17 22:37:15 controller.py:117] 
I 03-17 22:37:15 controller.py:117] == End of logs (ID: 199) ==
I 03-17 22:37:22 state.py:656] Cancellation skipped, job is already terminal
I 03-17 22:37:22 state.py:677] Cancellation skipped, job is not CANCELLING
I 03-17 22:37:24 controller.py:552] Killing controller process 69849.
I 03-17 22:37:24 controller.py:560] Controller process 69849 killed.
I 03-17 22:37:24 controller.py:562] Cleaning up any cluster for job 199.
I 03-17 22:37:24 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-2qjgobb6
<sky-payload>"<rich_init>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-2qjgobb6[/][/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-2qjgobb6[/][/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-2qjgobb6[/][/]</rich_start>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 22:37:26 storage.py:1505] [32mDeleted S3 bucket skypilot-filemounts-root-726f1974-2qjgobb6.[0m
I 03-17 22:37:26 controller.py:571] Cluster of managed job 199 has been cleaned up.
